### PR TITLE
Global

### PR DESCRIPTION
**Pull request by DennisTarik**

### CODE REVIEW

File: lib/footerComponent.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+export function createFooterComponent() {
+    const footerElement = createElement(
+        'footer',
+        {
+            className: 'footer',
+        },
+        [
+            createElement(
+                'h1', {
```
Comment (lajanzen) [inline lib/footerComponent.js]: I think an h1 does not really make sense in the footer, you would rather find it in the header element. Maybe use a p tag here? :)

```suggestion
                'p', {
```

File: main.js [hunk 2]
```diff
@@ -1,6 +1,20 @@
...
+
+  const headerComponent = createHeaderComponent();
+
+  const mainComponent = createMainComponent();
+
+  const footerComponent = createFooterComponent();
+
+  appElement.append(headerComponent);
+  appElement.append(mainComponent);
+  appElement.append(footerComponent);
```
Comment (lajanzen) [inline main.js]:
```suggestion
  appElement.append(headerComponent, mainComponent, footerComponent);
```

File: style.css [hunk 3]
```diff
@@ -1,3 +1,64 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
```
Comment (lajanzen) [inline style.css:1]:
```suggestion

```

File: style.css [hunk 4]
```diff
@@ -1,3 +1,64 @@
...
+  font-size: 4rem;
+}
+footer {
+  background-color: var(--color-footer);
+  text-align: center;
+  height: 20vh;
+  padding-top: 10rem;
+}
+
 #app {
```
Comment (lajanzen) [inline style.css:62]: Do you still need this app styling?

**Review by lajanzen — CHANGES_REQUESTED** [review]

**Review by Ziedlassoued — APPROVED** [review]
lgtm 👍

File: lib/footerComponent.js (hunk 1 above)
Comment (DennisTarik) [reply lib/footerComponent.js]: you are absolutly right 
thanks lara 🎈

File: style.css (hunk 3 above)
Comment (DennisTarik) [reply style.css:1]: ?

File: style.css (hunk 4 above)
Comment (DennisTarik) [reply style.css:62]: no i dont need it :D

File: style.css [hunk 5]
```diff
@@ -1,3 +1,64 @@
...
+  --font-family-text: 'Roboto', sans-serif;
+
+  /* Borders */
+  --border-radius: 1rem;
+}
+body {
+  font-family: var(--font-family-text);
+  margin: 0;
+  background-color: var(--color-background);
+  padding-top: 0;
```
Comment (lmachens) [inline style.css:30]: Is this required?

**Pull request merged**